README.md

```diff
@@ -73,19 +73,17 @@ Components have variant-property for different versions of the current component
 
```

Comment:
^^ Do we still have some static variants as stated above?

Reply:
And I did. From two almost identical descriptions, but not from here. Should be no fixed.